api/v1beta2/ocirepository_types.go

```diff
@@ -78,6 +78,10 @@ type OCIRepositorySpec struct {
 	// +optional
 	SecretRef *meta.LocalObjectReference `json:"secretRef,omitempty"`
 
+	// Verify specifies which provider to use to check whether OCI image is authentic.
```

Comment:
Maybe more around "how to verify...", as the field is composed of more than just the provider.

Reply:
that makes sense, and I've updated the comments based on your feedback.